lib/internal/streams/next-tick-browser.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+module.exports = function (cb) { 
+    var args = new Array(arguments.length - 1);
```

Comment:
it’s very slow to use new Array; why is this not slicing arguments into an array?

Reply:
Indeed. I updated it for performance and push-rebased: https://github.com/nodejs/readable-stream/pull/454/files#diff-773895e366710aaad05c2daf697abc214727c9de1227114b5694878bd33322b2